styles.css

```
@@ -27,4 +27,9 @@ body.format-multitopic.path-mod.pagelayout-incourse #page #page-content .activit
     display: none;
 }
 
+/* Course Index */
+body.format-multitopic .courseindex .courseindex-section.dropready .courseindex-item-content {
+    padding-bottom: 0;
+}
+
```

It looks like the padding needs to be added back in just below the activities list, to allow activities to be dragged to the end of the list.
```suggestion
body.format-multitopic.editing .courseindex .courseindex-section.dropready .courseindex-item-content .courseindex-sectioncontent {
    padding-bottom: 1em;
}

```